lib/pcb/convert-circuit-json-to-solder-paste-mask.ts

```diff
@@ -12,6 +12,7 @@ import { createSvgObjectsFromSolderPaste } from "./svg-object-fns/convert-circui
 import type { PcbContext } from "./convert-circuit-json-to-pcb-svg"
 import { DEFAULT_PCB_COLOR_MAP } from "./colors"
 import { getSoftwareUsedString } from "../utils/get-software-used-string"
+import { toNumeric } from "../utils/to-numeric"
```

Comment:
this drives me nuts `toNumeric` is ALWAYS A BUG!

we do not pass around numbers as strings ever! It's not in any spec! use `distance.parse` for parsing strings that may be represented w/ mm etc.! Anything else is going to break!